docs/build-aea-programmatically.md

```diff
@@ -46,106 +46,110 @@ OUTPUT_FILE = "output.txt"
 ## Create a private key
 We need a private key to populate the AEA's wallet.
 ``` python
-# Create a private key
-_create_default_private_key()
+    # Create a private key
+    _create_fetchai_private_key()
 ```
 
 ## Clearing the input and output files
 We will use the stub connection to pass envelopes in and out of the AEA. Ensure that any input and output text files are removed before we start.
 ``` python
-# Ensure the input and output files do not exist initially
-if os.path.isfile(INPUT_FILE):
-    os.remove(INPUT_FILE)
-if os.path.isfile(OUTPUT_FILE):
-    os.remove(OUTPUT_FILE)
+    # Ensure the input and output files do not exist initially
+    if os.path.isfile(INPUT_FILE):
+        os.remove(INPUT_FILE)
+    if os.path.isfile(OUTPUT_FILE):
+        os.remove(OUTPUT_FILE)
 ```
 
 ## Initialise the AEA
 We use the private key file we created to initialise a wallet, we also create the stub connection, tell it what Ledger APIs we are going to use (none in this example) and create a resources object containing skills, protocols and connections (this is initially empty). 
 
 Then we pass all of this into the AEA constructor to create our AEA.
 ``` python
-# Set up the wallet, stub connection, ledger and (empty) resources
-wallet = Wallet({FETCHAI: FETCHAI_PRIVATE_KEY_FILE})
-stub_connection = StubConnection(
-    input_file_path=INPUT_FILE, output_file_path=OUTPUT_FILE
-)
-ledger_apis = LedgerApis({'fetchai': {'network': 'testnet'}}, 'fetchai')
-resources = Resources()
-
-# Create an identity
-identity = Identity(name="my_aea", address=wallet.addresses.get(FETCHAI), default_address_key=FETCHAI,)
-
-# Create our AEA
-my_aea = AEA(identity, [stub_connection], wallet, ledger_apis, resources)
+    # Set up the Wallet, stub connection, ledger and (empty) resources
+    wallet = Wallet({FETCHAI: FETCHAI_PRIVATE_KEY_FILE})
+    stub_connection = StubConnection(
+        input_file_path=INPUT_FILE, output_file_path=OUTPUT_FILE
+    )
+    ledger_apis = LedgerApis({}, FETCHAI)
```

Comment:
why was the code changed?

Reply:
It is black reformatted. And because I am testing if the code blocks exist in the file we created to test it need to be identical. That means, I had to add the spaces.

Reply:
ok :)

Reply:
I meant, why was the ledger apis instantiation changed :)

Reply:
Ah, that's easy :D Because I copied the code from the toggle-able code and as it looks like I haven't changed the ledger_apis there :D Thanks for spotting :)